code/style.css

```diff
@@ -1,4 +1,189 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+/* Header */
+
+header {
+  background: wheat;
+  height: 150px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+
+header img {
+  float: none;
+  
+}
+
+nav {
+  display: none;
+}
+
+nav a {
+  color: black;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  text-decoration: none;
+  font-size: 20px;
+}
+
+.hamburger {
+  display: flex;
+  width: 100px;
+  height: 50px;
+  flex-direction: column;
+  justify-content: space-between;
+  
+}
+
+.hamburger div {
+  width: 50px;
+  height: 10px;
+  background: black;
+  border-radius: 5px;
+}
+
+/* body */
+
+body {
```

Comment:
Nice structure here too, so good to see. Try to keep the body styling at the top of the file since it wraps your content = good practise